From d5e62d0a0413ec65a4c8d9d8eb31b35a086320f0 Mon Sep 17 00:00:00 2001
From: Francois Gaffie <francois.gaffie@intel.com>
Date: Fri, 23 Nov 2012 15:56:28 +0100
Subject: Add Stream Flags key to setParameters

BZ: 70126
[CLEAN]

Audio Route Manager needs to know the type of stream to take the
appropriate routing decision (eg direct Output, MP3 Offload).
It must be define as key and filled by the policy.

Category: device enablement
Domain: AudioComms-Common
Origin: internal
Upstream-Candidate: no, needs rewok
Change-Id: Ie0267eece4693048de9852a03bdfc55e1296b536
Orig-Change-Id: I5d2273971ba8d279f3bbda1db811a39e99d3357f
Signed-off-by: Francois Gaffie <francois.gaffie@intel.com>
---
 include/hardware/audio.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 1aa2de2..07f7f12 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -107,6 +107,9 @@ __BEGIN_DECLS
 /* Screen state */
 #define AUDIO_PARAMETER_KEY_SCREEN_STATE "screen_state"
 
+/* Stream Flags */
+#define AUDIO_PARAMETER_KEY_STREAM_FLAGS "stream_flags"
+
 /* Always Listening/VTSV */
 #define AUDIO_PARAMETER_KEY_ALWAYS_LISTENING_STATUS "vtsv_active"
 #define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_ON "true"
-- 
1.7.4.1

